results.csv

```diff
@@ -1,18 +1,33 @@
-VehicleID,Delayed in hours
```

Comment:
Bei mir hat es funktioniert:
![grafik](https://user-images.githubusercontent.com/28202401/29243033-32647928-7f96-11e7-911c-f40204cba334.png)

Wenn man dann das Programm ausführt und sich dadurch `results.csv` wieder ändert erscheint sie bei mir nicht wieder in git als modified.
![grafik](https://user-images.githubusercontent.com/28202401/29243045-5e0b5c40-7f96-11e7-8f4d-e2303ae9e7f8.png)